service/discovery/extra/csaf/document_discover_test.go

```diff
@@ -0,0 +1,123 @@
+package csaf
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"clouditor.io/clouditor/v2/api/discovery"
+	"clouditor.io/clouditor/v2/api/ontology"
+	"clouditor.io/clouditor/v2/internal/crypto/openpgp"
+	"clouditor.io/clouditor/v2/internal/testutil/assert"
+	"github.com/csaf-poc/csaf_distribution/v3/csaf"
+)
+
+func Test_csafDiscovery_handleAdvisory(t *testing.T) {
+	type fields struct {
+		domain string
+		csID   string
+		client *http.Client
+	}
+	type args struct {
+		label    csaf.TLPLabel
+		file     csaf.AdvisoryFile
+		keyring  openpgp.EntityList
+		parentId string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantDoc assert.Want[*ontology.SecurityAdvisoryDocument]
+		wantErr bool
+	}{
+		{
+			name: "happy path",
+			fields: fields{
+				domain: goodProvider.Domain(),
+				csID:   discovery.DefaultCloudServiceID,
+				client: goodProvider.Client(),
+			},
+			args: args{
+				label: csaf.TLPLabelWhite,
+				file: csaf.HashedAdvisoryFile{
+					0: goodProvider.URL + "/.well-known/csaf/white/2020/some-id.json",
+				},
+				keyring: goodProvider.Keyring,
+			},
+			wantDoc: func(t *testing.T, got *ontology.SecurityAdvisoryDocument) bool {
+				// Some debugging output, that can easily be used in Rego
+				fmt.Println(ontology.ToPrettyJSON(got))
```

Comment:
Maybe use `log.debug` to reduce output when running all tests?

Reply:
ok

Reply:
Why are you logging the Ontology here? We do not need it here.